lib/blockscore/actions/retrieve.rb

```diff
@@ -11,8 +11,9 @@ module Actions
     # => #<BlockScore::Person:0x007fe39c424410>
     module Retrieve
       module ClassMethods
-        def retrieve(id)
-          get("#{endpoint}/#{id}", {})
+        def retrieve(id, options={}, lazy=true)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `lazy`. If it's necessary, use `_` or `_lazy` as an argument name to indicate that it won't be used.<br>Style/SpaceAroundEqualsInParameterDefault: Surrounding space missing in default value assignment.